### ISSUE TITLE
[BUG] Button in card automatically has expand="block" look

### ISSUE THREAD

**Issue by MortenGregersen** [kirbydesign/designsystem#352]
**Describe the bug**
When a KirbyButton is added to a KirbyCard it is shown as if it had a `expand="block"`.

**To Reproduce**

```html
<kirby-card>
  <Button kirby-button>Tap me</Button>
</kirby-card>
```

**Kirby version**
 - 0.0.179

**Expected behavior**
The button should only be as wide as its text - like shown [in the cookbook](https://cookbook.kirby.design/home/showcase/button).
![Screenshot 2019-06-13 at 09 25 06](https://user-images.githubusercontent.com/759680/59412176-288a5300-8dbd-11e9-8574-6324715b8f5b.png)

**Screenshots**
- Web:
![Screenshot 2019-06-13 at 09 13 13](https://user-images.githubusercontent.com/759680/59411721-2ecbff80-8dbc-11e9-88cb-59e6ba8a3127.png)
- iOS:
![Screenshot 2019-06-13 at 09 22 10](https://user-images.githubusercontent.com/759680/59411982-cc273380-8dbc-11e9-9819-33734c536299.png)
- Android:
![Screenshot 2019-06-13 at 09 22 18](https://user-images.githubusercontent.com/759680/59412015-d77a5f00-8dbc-11e9-8168-e7a4859da2e5.png)

**Desktop:**
 - OS: macOS Mojave
 - Browser: Chrome 75.0.3770.80 and Safari 12.1

**Native:**
 - iPhone6 (iOS 12.2) and Android Emulator (Android 9)

**Comment by jakobe** [comment]
Will be fixed by #341

**Comment by labanos** [comment]
Duplicate of #341
